security/tests.php

```diff
@@ -155,11 +155,13 @@ function rsssl_is_default_wp_prefix() {
  */
 function rsssl_has_admin_user() {
 
-	$users = get_users();
-	foreach ( $users as $user ) {
-		if ( $user->data->user_login === 'admin') {
-			return true;
-		}
+	global $wpdb;
+
+	$prepared = $wpdb->prepare("SELECT COUNT(*) FROM {$wpdb->prefix}users WHERE user_login = %s", 'admin');
```

Comment:
Voor performance een LIMIT 1 toevoegen aan de SQL query? Voor websites zoals ikonderneemhet.nl met 2000+ gebruikers.

https://stackoverflow.com/questions/455476/does-adding-limit-1-to-mysql-queries-make-them-faster-when-you-know-there-will

Reply:
Aangepast

Reply:
Check.